.github/workflows/increment-version.yml

```diff
@@ -0,0 +1,133 @@
+name: Increment version
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: Version to increment
+        required: true
+        default: auto
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  VERSION_FILE: ./package.json
```

Comment:
`package-lock.json` should be updated as well when we are updating version, otherwise with next `npm install` run we will have local diff like that:

![image](https://github.com/nimblehq/infrastructure-templates/assets/475367/5176ca95-cdd4-420c-afab-6d6782ae5f59)

Reply:
Fixed in https://github.com/nimblehq/infrastructure-templates/pull/237/commits/32130204673ce8e216c5dddd428d4632fa7498a2 🙏

Reply:
This is the new generated PR => https://github.com/nimblehq/infrastructure-templates/pull/263

Reply:
Great 🙂